src/pages/ProfileSettings/utils/handleInputsRender.tsx

```diff
@@ -1,14 +1,21 @@
-import { Fragment } from "react";
+import { Fragment, useContext } from "react";
+import { AuthContext } from "../../../contexts/Auth/AuthContext";
+
+export const HandleInputsRender = (arr: any): [] => {
+    const auth = useContext(AuthContext);
 
-export const handleInputsRender = (arr: any): [] => {
     return arr.map((element: any, index: any) => {
         return (
             <Fragment key={index}>
```

Comment:
Evitar usar index como key quando for fazer listas. 

Material de apoio explicativo: https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

Reply:
Tracei 2 estratégias para solucionar isso em todos os elementos renderizados por lista.

![image](https://github.com/SouJunior/vagas-frontend/assets/79734585/461925ca-8bf0-48e7-99f2-7d00fbff4636)
![image](https://github.com/SouJunior/vagas-frontend/assets/79734585/f487974b-6847-47d0-9f57-bd0336191ffb)

Adicionei um id ao dados e estou utilizando esse id como chave!